pkgs/by-name/ss/ssh-import-id/package.nix

```diff
@@ -0,0 +1,63 @@
+{
+  lib,
+  extraHandlers ? [ ],
+  fetchgit,
+  installShellFiles,
+  makeWrapper,
+  python3Packages,
+}:
+
+python3Packages.buildPythonApplication rec {
```

Comment:
Dont put python3Packages here, see https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#buildpythonapplication-function-buildpythonapplication-function for proper usage.

Reply:
Can you be more specify what you think is wrong here? 

https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#buildpythonapplication-function-buildpythonapplication-function says...

```bash
{ lib
, python3Packages
, fetchPypi
}:

python3Packages.buildPythonApplication rec {
  pname = "luigi";
  version = "2.7.9";
 [...]
 ```

Reply:
Wait, i confused things. I was reviewing as if this was a python-module but this is outside of pkgs/development/python-modules so buildPythonApplication should be fine. my bad.